extension/package.json

```
@@ -8,13 +8,15 @@
   },
   "devDependencies": {},
   "scripts": {
+    "build": "web-ext build && mv ../static/downloads/addon/latest/firefox_relay-*.zip ../static/downloads/addon/latest/firefox_relay.zip && npm run build:clean",
```

Trying to get away from `static/downloads`
```suggestion
    "build": "web-ext build && npm run build:clean",
```